example-app/SantokuApp/src/bases/ui/illustration/RemoveIllustration.tsx

```diff
@@ -0,0 +1,19 @@
+import {ColorProps, useResponsiveProp, useTheme} from '@shopify/restyle';
+import React from 'react';
+import {Path} from 'react-native-svg';
+
+import {StyledSvgIconBase, StyledSvgIconBaseProps} from '../common/StyledSvgIconBase';
+import {RestyleTheme} from '../theme/restyleTheme';
+
+export type DeleteIllustrationProps = StyledSvgIconBaseProps & ColorProps<RestyleTheme>;
+
+export const RemoveIllustration = ({color = 'white', size = 'p24', ...rest}: DeleteIllustrationProps) => {
```

Comment:
`DeleteIllustration` と `RemoveIllustration` を揃えておきましょう
```suggestion
export type RemoveIllustrationProps = StyledSvgIconBaseProps & ColorProps<RestyleTheme>;

export const RemoveIllustration = ({color = 'white', size = 'p24', ...rest}: RemoveIllustrationProps) => {
```

Reply:
20caed6d6789d0572d313df2934c5f6c80800f15 `refactor DeleteIllustrationProps -> RemoveIllustrationProps(commit suggestion)`